### ISSUE TITLE
[Feature Request] Display empty string

### ISSUE THREAD

**Issue by lndgalante** [wallabyjs/quokka#143]
### Issue description or question
When returning an empty string `''` Quokka doesn't display nothing so you don't really know what's happening.

### Sample code

```javascript
const firstNonRepeatingLetter = s => {
  const index = s
    .toLowerCase()
    .split('')
    .findIndex((char, _, s) => s.indexOf(char) === s.lastIndexOf(char))

  return index !== -1 ? s[index] : ''
}

firstNonRepeatingLetter('') //?
```

<img width="602" alt="screen shot 2018-01-05 at 10 31 16" src="https://user-images.githubusercontent.com/2475912/34611275-95e53cfa-f203-11e7-9fd6-1da5a58f0b4c.png">

### Code editor version

Visual Studio Code v1.19.1

### OS name and version

macOS High Sierra 10.13.2

**Comment by ArtemGovorov** [comment]
The feature is implemented in v1.0.91.

**Comment by lndgalante** [comment]
Thanks @ArtemGovorov !

<img width="625" alt="screen shot 2018-01-15 at 01 57 48" src="https://user-images.githubusercontent.com/2475912/34927785-87bc781e-f997-11e7-82b0-5752ac73368f.png">
